tests/integration/conftest.py

```diff
@@ -98,3 +99,10 @@ def _get_osname():
             if line.startswith("PRETTY_NAME="):
                 return line.split("=", maxsplit=1)[1].strip().strip('"')
     return ""
+
+
+def download_NM_plugin(plugin):
+    cmdlib.exec_cmd(("dnf", "install", "-y", "--downloadonly", f"{plugin}",))
+
+
+download_NM_plugin("NetworkManager-team")
```

Comment:
If you meant to call it before the tests, it fits as a session fixture.

Reply:
Done! Thanks!